torch/_functorch/aot_autograd.py

```diff
@@ -1135,6 +1138,16 @@ def forward(*runtime_args: tuple[Any]):
     forward.named_parameters = mod.named_parameters
     forward.named_buffers = mod.named_buffers
 
+    # Add a serialize function
+    def grab_serialize_fn(fn):
+        if isinstance(fn, SerializableCompiledFunction):
+            return fn.serialize_fn
+        elif hasattr(fn, "__wrapped__"):
+            return grab_serialize_fn(fn.__wrapped__)
+        else:
+            return None
```

Comment:
do you think it would be too aggressive for this to raise an error? (as long as caching is turned on, shouldn't we expect the serialize_fn to always exist?)